packages/react-native-win32/src/Libraries/Utilities/FocusManager.win32.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+import { findNodeHandle, NativeModules } from 'react-native';
+
+/*
+** This is a helper class intended to allow usage of Polite/Aggressive Focus for win32.
+*/
+class FocusManager {
+
+    // This function takes in a ref and a bool value. If setWindowFocus = true, call aggressive focus.
+    // Else, call polite focus
+    static Focus(ref, setWindowFocus)
+    {
+        if(ref)
+        {
+            if(setWindowFocus)
+            {
+                NativeModules.UIManager.dispatchViewManagerCommand(
+                    findNodeHandle(ref),
+                    NativeModules.UIManager.getViewManagerConfig('RCTView').Commands.aggressivefocus,
```

Comment:
>'RCTView' [](start = 65, length = 9)

(nit) it would be better to encode this string in a string const, and then reuse it

Reply:
This is a bit undocumented, but you can also 
 replace `NativeModules.UIManager.getViewManagerConfig('RCTView').Commands.aggressivefocus` with just `'aggressivefocus'` if you want. ViewManager commands for RN were internally all moved to raw strings, but grabbing the command constant from the native ViewManager config is still the public way to do this.